docs/src/create-apps/multi-app.md

```diff
@@ -101,47 +107,65 @@ In this setup, your app configuration has to be in the top repository, not in a
 In this case, and any other case where you want the configuration to be separate from the code,
 you need to specify the code root in the configuration file.
 
-So your project repository might look like this:
+So your [project repository](https://github.com/platformsh-templates/bigfoot-multiapp/tree/multiapp-submodules-subfolders) might look like this:
 
 ```text
-├── app1
-│   ├── .platform.app.yaml  <- App 1 configuration
-│   └── app1-submodule
-|       └── ...             <- App 1 code from submodule
-├── app2
-│   ├── .platform.app.yaml  <- App 2 configuration
-│   └── app2-submodule
-|       └── ...             <- App 2 code from submodule
+├── .platform
+│   └── routes.yaml
+├── admin
+│   ├── .platform.app.yaml  <- API Platform Admin app configuration
+│   └── admin-submodule
+│       └── ...             <- API Platform Admin code from submodule
+├── api
+│   ├── .platform.app.yaml  <- Bigfoot app configuration
+│   └── api-submodule
+│       └── ...             <- Bigfoot code from submodule
+├── gatsby
+│   ├── .platform.app.yaml  <- Gatsby app configuration
+│   └── gatsby-submodule
+│       └── ...             <- Gatsby code from submodule
+└── mercure
+│   ├── .platform.app.yaml  <- Mercure Rocks app configuration
+│   └── mercure-submodule
+│       └── ...             <- Mercure Rocks code from submodule
 └── .gitmodules
-└── .platform
-    └── routes.yaml
 ```
 
-Your `.gitmodules` file would define both submodules:
+Your `.gitmodules` file would define all submodules:
 
 ```txt {location=".gitmodules"}
-[submodule "app1"]
-    path = app1/app1-submodule
-    url = https://github.com/your-org/app1
-[submodule "app2"]
-    path = app2/app2-submodule
-    url = https://github.com/your-org/app2
+[submodule "admin-submodule"]
+	path = admin/admin-submodule
+	url = https://github.com/platformsh-templates/bigfoot-multiapp-admin.git
+	branch = without-platform-app-yaml
```

Comment:
I would suggest the branch names for the submodules be simpler. If we aren't going to point to a typical default branch name (e.g. `main`, `master`), then we should explain why, or choose a name that doesn't give the impression the branch should _not_ contain a platform config file.

Reply:
In my Bigfoot's submodule repos, i name branches regarding the difference between them (`with-platform-app-yaml` and `without-platform-app-yaml`) 
And as in this structure example, the `.platform.app.yaml` file is in the main repo, i name the submodule branch like this reflecting the fact there is no `.platform.app.yaml` file inside the submodule repo (in this case)
But you're right, there is no example in the multi-app page where we show that we could define the submodule, containing the `.platform.app.yaml` file : 
in [this branch](https://github.com/platformsh-templates/bigfoot-multiapp/tree/main), the submodule are added at the root of the main project and contain `.platform.app.yaml` file (submodule branch `with-platform-app-yaml`).